cyberark/secretless-broker
# Mobile nav fixes, removed search, and updated google tag manager

**What does this PR do?**
1. On mobile screens, open dropdown menus in the top nav are overlapping other items in the navigation.
- Closes #476 
- Closes #352 
2. Fixes errors due to multiple Google tag manager & analytics instances
3. Removes WIP search functionality

**How should I test?**
1. Open web inspector, and confirm there are no errors pertaining to google tag manager (should be no errors for anything)
2. Confirm that the magnifying glass icon is removed from the top nav, and manually navigating to /search does not go to a search page
3. Confirm that when viewing at mobile screen widths, opening the dropdown menu does not overlap other navigation items

**Screenshots**
Before (nav items are hidden under blue drop down menu):
![screenshot_2018-11-06 secretless broker secretless 1](https://user-images.githubusercontent.com/123787/48090167-36974600-e1d4-11e8-8c37-9c44b44bf3da.png)

After (happy happy joy joy):
![screenshot_2018-11-06 secretless broker secretless](https://user-images.githubusercontent.com/123787/48090204-4ca50680-e1d4-11e8-94cb-4eda3240f10e.png)

Search removed
Search icon was previously in top nav. This was only appearing in Staging
![screenshot_2018-11-09 community secretless](https://user-images.githubusercontent.com/123787/48274900-5025d100-e412-11e8-9b2f-f6a2753dc065.png)
